Fixes bug reported at debian bugzilla:
http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=375365

How to reproduce:
$ tftp
tftp> connect
(to) 
Segmentation fault

Just pressing enter at the "(to)" prompt crashes tftp.

--- ./tftp/main.c.orig	2007-07-19 11:31:55.000000000 +0400
+++ ./tftp/main.c	2007-07-19 12:54:25.000000000 +0400
@@ -192,12 +192,17 @@
 {
 	struct hostent *host;
 	size_t len;
+	char *retval;
 
 	if (argc < 2) {
 		strcpy(line, "Connect ");
 		printf("(to) ");
 		len = strlen(line);
-		fgets(line+len, sizeof(line)-len, stdin);
+		retval = fgets(line+len, sizeof(line)-len, stdin);
+		if (isspace((int)*retval)) {
+			printf ("No value supplied. Connecting to localhost.\n");
+			strcpy(line+len, "localhost\n");
+		}
 		makeargv();
 		argc = margc;
 		argv = margv;
